### DISCUSSION TITLE
Support for int8 matrix multiplication

### DISCUSSION THREAD

**Discussion by hchoi-moveworks** [ELS-RD/kernl#287] [General]
Would there be a support for optimizing model that leverages int8 matmul[1] ? 

[1]: https://huggingface.co/blog/hf-bitsandbytes-integration

**Comment by pommedeterresautee** [comment]
This work is ongoing. Perf is good but not yet on par with cutlass (like 10-20% slower). We are working on a full process to support quantization out of the box.
